src/pages/en/guides/deploy/aws.md

```diff
@@ -0,0 +1,226 @@
+
+    ```json
+    {
+      "Version": "2012-10-17",
+      "Statement": [{
+        "Effect": "Allow",
+        "Principal": {
+          "AWS": "arn:aws:iam::cloudfront:user/CloudFront Origin Access Identity <CLOUDFRONT_OAI_ID>"
+        },
+        "Action": "s3:GetObject",
+        "Resource": "arn:aws:s3:::astro-aws/*"
+      }]
+    }
+    ```
+
+    :::caution
+    Do not forget to replace `<CLOUDFRONT_OAI_ID>` with the name of your CloudFront Origin Access Identity ID. You can find the CloudFront Origin Access Identity ID in **CloudFront > Origin access identities** after setting up CloudFront.
+    :::
+
+### CloudFront setup
+
+1. Create a CloudFront distribution with the following values:
+    * **Origin domain:** Your S3 bucket
+    * **S3 bucket access:** "Yes use OAI (bucket can restric access to only CloudFront)"
+    * **Origin access identity:** Create a new origin access identity
+    * **Viewer - Bucket policy:** "No, I will update the bucket policy"
+    * **Viewer protocol policy:** "Redirect to HTTPS"
+    * **Default root object:** `index.html`
+
+This configuration will block access to your S3 bucket from the public internet and serve your site using the global CDN network. You can find your CloudFront distribution URL in the bucket's **Distributions > Domain name**.
+
+### CloudFront Functions setup
+
+Unfortunately, CloudFront does not support multi-page `sub-folder/index` routing by default. To configure it, we will use CloudFront Functions to point the request to the desired object in S3.
+
+1. Create a new CloudFront function with the following code snippet. You can find CloudFront functions in **CloudFront > Functions**.
+
+    ```js
+    function handler(event) {
+      var request = event.request;
+      var uri = request.uri;
+
+      // Check whether the URI is missing a file name.
+      if (uri.endsWith('/')) {
+        request.uri += 'index.html';
+      } 
+      // Check whether the URI is missing a file extension.
+      else if (!uri.includes('.')) {
+        request.uri += '/index.html';
+      }
+
+      return request;
+    }
+    ```
+  2. Attach your function to the CloudFront distribution. You can find this option in your CloudFront distribution's settings > Behaviors > Edit > Function associations.
+        * **Viewer request - Function type:** CloudFront Function.
+        * **Viewer request - Function ARN:** Select the function you created in the previous step.
+
+## Continuous deployment with GitHub Actions
+
+There are many ways to set up continuous deployment for AWS. One possibility for code hosted on GitHub is to use a [GitHub Actions](https://github.com/features/actions) to deploy your website every time you push a commit.
+
+1. Create a new policy in your AWS account using [IAM](https://aws.amazon.com/iam/) with the following permissions. This policy will allow you to upload build files to your S3 bucket and invalidate the CloudFront distribution files when you push a commit.
```

Comment:
```suggestion
1. Create a new policy in your AWS account using [IAM](https://aws.amazon.com/iam/) with the following permissions. This policy will allow you to upload built files to your S3 bucket and invalidate the CloudFront distribution files when you push a commit.
```
I'm not sure this is wrong, and you might mean "build" files, but you've previously used "built" (has been built) files. So, just pointing this out!